reminders/script.js

```diff
@@ -1,6 +1,70 @@
+var toDoList = getToDos() || [];
+
+
 function createItem() {
-    var list = document.getElementById("list-items").value;
-    var todo = document.getElementById("todo");
-    todo.insertAdjacentHTML('beforeend', '<input type="checkbox"> ' + list + '<br>');
-    console.log(list)
-}
\ No newline at end of file
+    var newToDo = addNewTodo();
+    updateTodoList(newToDo);
+    saveTodosLocally();
+    addToDoToHTML(newToDo);
+    console.log(toDoList);
```

Comment:
this is looking a lot cleaner, each line tells me something distinct. small nitpick: move the `newTodo` lines together since they're related (easier to understand IMO)